proposals/dao/tip_121b.ts

```diff
@@ -36,10 +41,13 @@ Steps:
 const fipNumber = 'tip_121b';
 
 const dripPeriod = 3600; // 1 hour
-const dripAmount = ethers.utils.parseEther('2500000'); // 2.5m Fei
+const dripAmount = ethers.utils.parseEther('1000000'); // 1m Fei
```

Comment:
Why are we lowering the `dripAmount`? Biggest EOA has just under $2m to claim? We support partial redemptions, but probably a better user experience for those claimants if dripAmount > $2m?

Reply:
Because a drip amount of 2m doesn't provide enough protection. Easy enough to walk those two users through a partial